### DISCUSSION TITLE
Cann't renew my student developer pack

### DISCUSSION THREAD

**Discussion by zlysm** [community/community#67272] [GitHub Education] [answered]
### Hi everyone,

Recently I was unable to access github copilot and finally I found out that my student developer pack expired. When I resubmit, it says `Hi, Yiming Zhang! You were last verified as a student on. It is not necessary for you to re verify at this time.`

But at the bottom of the submission record, it said `Please submit a new request to apply for a discount.` This troubles me because I submitted it many times and was prompted that I don’t need to re verify, but my student pack really can’t be used. For example, I can’t use copilot, there is no pro logo on the homepage, etc.

Hope someone can help me

**Comment by Wenrh2004** [comment]
Yes, I'm experiencing the same problem and added the following to the details, when we select the email we used for our last academic certification on the benefits page, he suggests that our account is still on benefits, and the status on the right side shows that we need to resubmit the application, and the continue at the bottom is not clickable as well as the main branch protection is not working, please tell me if you are having this problem as well What about.

**Reply by zlysm** [reply]
The `continue` button is clickable for me, but after clicking it, an error message says `Hi, Yiming Zhang! You were last verified as a student on. It is not necessary for you to re verify at this time.`

**Reply by Wenrh2004** [reply]
This student certification really makes me hesitate to host and collaborate on my team's code through GitHub, his experience is really bad, I used to be a staunch defender of GitHub, but now I have to reconsider the team code collaboration and hosting platform.

**Reply by Wenrh2004** [reply]
<img width="1304" alt="截屏2023-09-17 00 49 32" src="https://github.com/community/community/assets/122615371/3929eca8-1f11-48e6-8eb7-eb691b1ceed6">
So amazing！

**Reply by samus-aran** [reply]
Hi @Wenrh2004 

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there [(which I know you have already)](https://github.com/orgs/community/discussions/67326). 👍

**Comment by FanBB2333** [comment]
The same problem, unhandled yet.

**Comment by sinne903** [comment]
I had this problem too and still don't know how to fix it

**Comment by samus-aran** [comment, marked as answer]
Hi @zlysm 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well. We often reverify yearly or sooner if required.

It sounds like there is an issue between your last verification date and now. Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by zlysm** [reply]
problem solved, thanks!

**Reply by praticodes** [reply]
@samus-aran Hi, I'm having the same issue. Would you be able to help with this?

**Reply by romain22222** [reply]
Hi, I'm also having the same issue. Is there a way I can contact you for further help (my coupon expires at the end of month, and the application form won't go to the next step, while having position tracking on...)

My bad, it's just that Brave isn't a supported browser while being a Chromium based browser...

**Comment by CharlesBaudinet** [comment]
Hi @samus-aran, I have the same issue. How can I solve it ? 

Best regards,
Charles
